getting-started/installation.md

```diff
@@ -141,10 +147,32 @@ let config = {
 };
 ```
 
+#### Start MagicMirror² in Server mode on boot
+
+To always start MagicMirror² in server only mode you need to edit the `installers/mm.sh` file otherwise it will start in full mode after a reboot:
+
+1. Backup mm.sh: `cp installers/mm.sh installers/mm.sh.bak`
```

Comment:
@rejas don't know anything about this `mm.sh` stuff, totally irrelevant when using docker, so is this a standard setup or how should we document it here?
Found in `mm.sh` this line
```bash
# This file is still here to keep PM2 working on older installations.
```

Reply:
the `mm.sh` file is removed on `develop` so `installers/mm.sh` will not exist anymore with next release. This file will be part of sam's script repo.

Reply:
yes, please remove the stuff about mm.sh